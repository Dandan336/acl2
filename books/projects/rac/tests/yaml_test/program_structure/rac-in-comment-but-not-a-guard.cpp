// RAC begin

// Rac: this is a comment not a guard

int foo()
{
  return 2;
}

// RAC end
