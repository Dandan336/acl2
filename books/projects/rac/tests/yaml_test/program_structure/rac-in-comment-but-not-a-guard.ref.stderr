rac-in-comment-but-not-a-guard.cpp:3: syntax error, unexpected RAC
